packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -442,7 +442,12 @@ describe("Presence", () => {
 						});
 					}
 
-					describe("when local client disconnects", () => {
+					// When local client disconnects, we lose the connectivity status updates for remote attendees in the session.
+					// Upon reconnect, we mark all remote attendees connections as "stale".
+					// Remote attendees with stale connections are given 30 seconds after local reconnection to show signs of life
+					// before their connection status set to "Disconnected".
+					// If an attendee with a stale connection becomes active, their "stale" status is removed.
```

Comment:
So, the only way an attendee can become active once we reconnect is if they send some signal? Why would they send a signal if they think they have maintained their connection?

Reply:
I think I don't understand "...if they think they have maintained their connection". If a client notices another client disconnected, they should know that the remote client will not be aware of their future connection status unless they can prove it (datastore update, join/ join response where the sender, collateral information where we see they're in audience, etc.).

Reply:
We don't have anything in the protocol [for non-targeted case] to do that. There is no case that says when a client sees a join (in these cases that would be the join the local client sends) they must respond.
Here is the thing that I think is overlooked, if the remote client is in the audience, then they aren't stale/disconnected.

Reply:
Right okay yes. I wrote in the edit above a client being in audience would consider them active. I'll reword description to make this clear.

Reply:
Changed sentence in description to: " Remote attendees with stale connections are given 30 seconds after local reconnection to prove they are connected (e.g. being in audience, sending an update, or (re)joining the session) before their connection status is set to 'Disconnected' "